apps/dapp-console/app/helpers/apiClient.ts

```diff
@@ -6,15 +8,28 @@ import type { ApiV0 } from '@eth-optimism/dapp-console-api'
 
 import { ENV_VARS } from '@/app/constants/envVars'
 
+function parseAccessToken(accessToken: string) {
+  return JSON.parse(accessToken)
```

Comment:
this removes the extra quotes and slashes that are getting added in localStorage